docs/tutorials/HD143006_Part_2.py

```diff
@@ -0,0 +1,481 @@
+#
+# This tutorial is a continuation of the [HD143006 Part 1](HD143006_Part_1.html) tutorial. It covers the same content as the MPoL tutorials on [Optimization](optimization.html), [Initalizing with the Dirty Image](initializedirtyimage.html), and [Cross Validation](crossvalidation.html) but in a streamlined fashion and using real data. These other tutorials provide a more comprehensive breakdown of each step in this tutorial.
+#
+# This tutorial will be going through how to initialize the model, the imaging and optimization process, how to use cross validation to improve the choice of hyperparameters in the model to more accurately predict new data, and how to analyze the results of our work with TensorBoard.
+#
+# ### Loading Data
+# Let's load the data as we did in the previous HD143006 tutorial ([Part 1](HD143006_Part_1.html)) and create the MPoL Gridder object.
+#
+# *You can either download these two files (HD143006_continuum.fits and HD143006_continuum.npz) directly to your working directory, or use Astropy to download them during run time.*
+
+# + cell_id="00001-be94721b-eee2-4e2e-96e2-dc65b9fd4f5b" deepnote_cell_type="code" deepnote_to_be_reexecuted=false execution_millis=663 execution_start=1623447169390 source_hash="4f0f20f8" tags=[]
+from astropy.io import fits
+import matplotlib.pyplot as plt
+import numpy as np
+from astropy.utils.data import download_file
+from torch.utils import tensorboard
+
+# downloading fits file
+fname_F = download_file(
+    "https://almascience.nrao.edu/almadata/lp/DSHARP/images/HD143006_continuum.fits",
+    cache=True,
+    pkgname="mpol",
+)
+# downloading extracted visibilities file
+fname_EV = download_file(
+    "https://zenodo.org/record/4904794/files/HD143006_continuum.npz",
+    cache=True,
+    pkgname="mpol",
+)
+# load extracted visibilities from npz file
+dnpz = np.load(fname_EV)
+uu = dnpz["uu"]
+vv = dnpz["vv"]
+weight = dnpz["weight"]
+data = dnpz["data"]
+# opening the fits file
+dfits = fits.open(fname_F)
+cdelt_scaling = dfits[0].header["CDELT1"] * 3600  # scaling [arcsec]
+cell_size = abs(cdelt_scaling)  # [arcsec]
+# close fits file
+dfits.close()
+
+# + cell_id="00008-d56e3afe-45cf-4fe5-a4e7-1803f28deec4" deepnote_cell_type="code" deepnote_to_be_reexecuted=false execution_millis=27 execution_start=1623441758279 source_hash="b76fed2d"
+from mpol import gridding, coordinates
+
+# creating Gridder object
+coords = coordinates.GridCoords(cell_size=cell_size, npix=512)
+gridder = gridding.Gridder(
+    coords=coords,
+    uu=uu,
+    vv=vv,
+    weight=weight,
+    data_re=data.real,  # separating the real and imaginary values of our data
+    data_im=data.imag,
+)
+
+# -
+
+#
+# ### Getting the Dirty Image and Creating the Model
+#
+# First, we are going to get the dirty image from our Gridder object. We will use the Briggs weighting scale and set `robust=0.0` here as this option leads to a dirty image resembling the DSHARP CLEAN image (see [Part 1](HD143006_Part_1.html)).
+
+import torch
```

Comment:
Fairly minor point, but I think you should move `import torch` and also `dirty_image = torch.tensor(img.copy())` to after you say "We now have everything from the last tutorial loaded and we can create the model." Since you didn't mention tensors/pytorch in part 1, I think it would read better if you introduced this aspect of it after saying we are caught up from part 1.